### ISSUE TITLE
NTR: Liver biopsy: glycogen phosphorylase a activity low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9075]
New term request Liver biopsy: glycogen phosphorylase a activity low (segal_050203152635): Hepatic biopsy

**Comment by pnrobinson** [comment]
@MickeySegal

is this the same activity as
[Reduced muscle glycogen phosphorylase activity HP:6000358](https://hpo.jax.org/app/browse/term/HP:6000358)?
If so, we should rename the term Reduced tissue glycogen phosphorylase activity and give it the two narrow synonyms Muscle and Liver...activity, because I think that the resultts would be clinically equivalent.

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK5941/
"The three [isoforms](https://www.ncbi.nlm.nih.gov/books/n/gene/glossary/def-item/isoforms/) of glycogen phosphorylase – muscle, liver, and brain – are encoded by different genes.
  Liver glycogen phosphorylase, encoded by PYGL"

So since all 3 are encoded by different genes it seems reasonable to consider them different and have a separate entry for each.  The muscle form is encoded by PYGM and is abnormal in McArdle disease.  The brain form is encoded by PYGB but has no known disease association.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000979
- label: Reduced hepatic glycogen phosphorylase activity
- parents: [Abnormal liver enzyme activity or concentration (HP:0034685)](https://hpo.jax.org/app/browse/term/HP:0034685)
- pmid: 
- def: Activity or concentration of glycogen phosphorylase L in the liver below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9075
- synonyms:
